releases/protocol/release-specs-v1.7.14-Interim.md

```diff
@@ -0,0 +1,124 @@
+[comment]: <> (tags/v1.7.16)
+
+# Contents
+
+This document explains the contents of the rc/v1.7.next1 release codenamed Interim. It is split in 2 sections:
+- the **features** list containing detailed insights of the feature along with the external impact and the relevant
+  pull requests list
+- the **smaller features and fixes** area contains the one-pull request small features or fixes along with the
+  external impact details
+
+This documentation is relevant for the `tags/v1.7.16` tag release.
+
+# Features
+
+## 1. Relayed v3 [#5741](https://github.com/multiversx/mx-chain-go/pull/5741)
+
+Relayed v3 feature brings a cheaper, improved version of relayed transactions, that can hold multiple inner transactions. 
+The inner transactions will be full transactions.
+
+With this feature, a fix on the base cost of relayed transactions was implemented, to follow the minimum gas rule.
```

Comment:
_Additionally_. 

Also, maybe it should be specified that this additional fix affects all types of relayed transactions?